generator/src/test/java/com/scottlogic/deg/generator/cucumber/If.feature

```diff
@@ -1668,7 +1668,7 @@ Scenario: Running an if request that contains a non contradictory aValid constra
        | "ccc"          | "GB0002634946" |
        | "dddd"         | "GB0002634946" |
 
-@ignore
+@ignore #issue 246
```

Comment:
I think this has been resolved now? If not can you create a new issue as #246 is closed.

Reply:
I am altering this as part of ticket #501 - is that OK?

Reply:
Yes, fine